src/app/public/modules/date-range-picker/date-range-picker.component.ts

```diff
@@ -178,6 +180,20 @@ export class SkyDateRangePickerComponent
   @Input()
   public label: string;
 
+  /**
+   * Indicates whether to mark the start date input as required.
+   * @default false
+   */
+  @Input()
+  public startDateRequired: boolean = false;
+
+  /**
+   * Indicates whether to mark the end date input as required.
```

Comment:
```suggestion
   * Indicates whether to require users to specify an end date.
```
Same assumption as for start date.

Reply:
Done.